### DISCUSSION TITLE
SURFACE OXIDATION MODEL FOR CHAR OXIDATION

### DISCUSSION THREAD

**Discussion by shanmugasundaram2706** [firemodels/fds#13115] [Q&A]
Hai Developers,
                 Could you please clarify about the calculation of the char oxidation rate if the "SURFACE OXIDATION MODEL" is kept "FALSE"?
How will the FDS calculate oxygen mass fraction at the fuel surface? 

Thanks in advance,

Regards,
Shanmugasundaram D

**Comment by ericvmueller** [comment]
This is an area of active development, but currently the approach is described by equations 9.2 and 9.3 of the User's Guide. The oxygen volume fraction in the gas phase cell is used at the solid surface, and it is assumed to decay in depth inside the solid based on the `GAS_DIFFUSION_DEPTH`.

**Comment by shanmugasundaram2706** [comment]
Dear Eric Mueller,
             Thanks for your reply and this is very helpful.

Regards,
Shanmugasundaram D
